src/GNDStk/Tree/src/make.hpp

```diff
@@ -0,0 +1,96 @@
+
+// -----------------------------------------------------------------------------
+// Functions for MAKING or MODIFYING trees
+// -----------------------------------------------------------------------------
+
+// init(top-level node, format[, version[, encoding]])
+// Example:
+//    GNDStk::Tree<> t(GNDStk::child::reactionSuite,GNDStk::format::xml);
+// Briefly:
+//    Tree<> t(reactionSuite,format::xml);
+// with appropriate usings.
+
+
+
+// ------------------------
+// top, format, ...
+// ------------------------
+
+template<class RESULT, bool MULTIPLE, class METADATA, class CHILDREN>
+nodeType &init(
```

Comment:
These methods should also be in a different file to strictly follow the style guide.

Reply:
There's an init.hpp now.